cookbook/assistants/add_messages.py

```diff
@@ -0,0 +1,9 @@
+from phi.assistant import Assistant
+from phi.llm.openai import OpenAIChat
+
+Assistant(
+    llm=OpenAIChat(model="gpt-3.5-turbo", stop="</answer>"),
+    system_prompt="What is the color of a banana? Provide your answer in the xml tag <answer>.",
+    add_messages=[{"role": "assistant", "content": "<answer>"}],
```

Comment:
```suggestion
    additional_messages=[{"role": "assistant", "content": "<answer>"}],
```

Maybe a mypy type check would catch this bug.